README.md

```diff
@@ -1,5 +1,11 @@
 
+
+# nmrec <a href='https:/metrumresearchgroup.github.io/nmrec'><img src='man/figures/logo.png' align="right" /></a>
```

Comment:
Just a note: this href is nice on the GitHub README side because it provides a link to the site.  On the site side, it actually ends up as an invalid link (<https://metrumresearchgroup.github.io/metrumresearchgroup.github.io/nmrec>) that redirects to <https://metrumrg.com/>.  (bbr's logo has the same behavior.)

Given that 1) the GitHub-side redirect is nice and 2) it's not useful for the logo on the site side to link to the current page, that redirect behavior is fine, though it may be worth looking into how to avoid the invalid link at some point.